docs/design/jet/025-jet-job-partition-pruning.md

```diff
@@ -121,6 +123,26 @@ partitions and for all relations. Then, we extract required partitions set to ru
 baked by `__sql.requiredPartitions` argument. This indicates that `ExecutionPlanBuilder` will
 try to choose only the required members.
 
+##### Mixed approach
+
+In addition to SQL-oriented approach, described above, we can attach the DAG analysis in
+`ExecutionPlanBuilder` during required members calculation to quickly determine required members to run the job.
+We are interested on it for two cases:
+
+- support `allToOne` edge : all edges are directed to one member and partitioned by single key.
+  These partitions and members (if multiple) are also a subjects to be included into the DAG result.
+  Main motivation here is that most of the queries highly likely will have `allToOne` to forward
+  scanned data to the coordinator, and we want to calculate it in proper place in less error-prone way.
```

Comment:
`allToOne` support has 2 elements: including member (coordinator) and including appropriate partition id in the job. It is unclear from the description

Reply:
https://github.com/hazelcast/hazelcast/pull/25211/commits/df0bfa228e8ff8539fa9a9c4c1292ce7a123c2b2